app/export.go

```diff
@@ -224,3 +227,14 @@ func Setup(isCheckTx bool) *AkashApp {
 
 	return app
 }
+
+func OptsWithGenesisTime(seed int64) servertypes.AppOptions {
+	r := rand.New(rand.NewSource(seed)) // nolint: gosec
+	genTime := simulation.RandTimestamp(r)
+
+	appOpts := viper.New()
```

Comment:
What is the rationale behind creating our own viper instance here?

Reply:
`NewApp` needs an instance of `servertypes.AppOptions` interface.
Viper implements this interface, so using it for testing.